packages/api-explorer/src/params.css

```diff
@@ -15,4 +15,16 @@
     -moz-user-select: none;
     -ms-user-select: none;
     user-select: none;
-}
\ No newline at end of file
+}
+
+label {
```

Comment:
the label required style is already handled. 
In this css file i add a required stars feature

Reply:
class `required` is already applied by jsoneditor, am I right?

Reply:
yes